lib/MadNLPTests/src/MadNLPTests.jl

```diff
@@ -331,6 +331,27 @@ function eigmina(optimizer_constructor::Function; Arr = Array)
     end
 end
 
+function lp(optimizer_constructor::Function; Arr = Array)
```

Comment:
I would maybe add a comment referring to the github issue to explain why we have this test. How about renaming the function `lp_issueXXX` with `XXX` the id of the github issue?